src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/ResourceGroupDeployments/TestAzureResourceGroupDeploymentCmdlet.cs

```diff
@@ -35,6 +35,9 @@ public class TestAzureResourceGroupDeploymentCmdlet : ResourceWithParameterCmdle
         [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The deployment mode.")]
         public DeploymentMode Mode { get; set; }
 
+        [Parameter(Mandatory = false, ValueFromPipelineByPropertyName = true, HelpMessage = "The name of a deployment to roll back to on error, or use as a flag to roll back to the last successful deployment.")]
+        public string RollbackAction { get; set; }
```

Comment:
Why would this go under test deployment?  How does this impact a deployment test?

Reply:
deployment validation will check for the validity of the specified on error deployment